examples/server/main.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/functionalfoundry/graphqlws"
+	"github.com/graphql-go/graphql"
+	"github.com/graphql-go/handler"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type Document struct {
+	Title   string `json:"title"`
+	Content string `json:"content"`
+}
+
+var documents = []Document{
+	{Title: "My diary", Content: "Today I had fun with graphqlws"},
+	{Title: "Todo", Content: "Add a complete example"},
+}
+
+var schema graphql.Schema
+var subscriptionManager graphqlws.SubscriptionManager
+
+func main() {
+	log.SetLevel(log.InfoLevel)
+	log.Info("Starting example server on :8085")
+
+	var documentType = graphql.NewObject(
+		graphql.ObjectConfig{
+			Name: "Document",
+			Fields: graphql.Fields{
+				"title": &graphql.Field{
+					Type: graphql.String,
+				},
+				"content": &graphql.Field{
+					Type: graphql.String,
+				},
+			},
+		},
+	)
+
+	var queryType = graphql.NewObject(
+		graphql.ObjectConfig{
+			Name: "Query",
+			Fields: graphql.Fields{
+				"document": &graphql.Field{
+					Type: documentType,
+					Args: graphql.FieldConfigArgument{
+						"docId": &graphql.ArgumentConfig{
+							Type: graphql.Int,
+						},
+					},
+					Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+						docID := p.Args["docId"].(int)
+						return documents[docID], nil
+					},
+				},
+			},
+		},
+	)
+
+	var mutationType = graphql.NewObject(
+		graphql.ObjectConfig{
+			Name: "Mutation",
+			Fields: graphql.Fields{
+				"updateDocument": &graphql.Field{
+					Type: documentType,
+					Args: graphql.FieldConfigArgument{
+						"docId": &graphql.ArgumentConfig{
+							Type: graphql.Int,
+						},
+						"title": &graphql.ArgumentConfig{
+							Type: graphql.String,
+						},
+						"content": &graphql.ArgumentConfig{
+							Type: graphql.String,
+						},
+					},
+					Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+
+						docID := p.Args["docId"].(int)
+						documents[docID].Title = p.Args["title"].(string)
+						documents[docID].Title = p.Args["title"].(string)
+
+						for _, subscriptions := range subscriptionManager.Subscriptions() {
+							for _, subscription := range subscriptions {
+
+								// JSON interface is float64
+								var subdocID int = int(subscription.Variables["docId"].(float64))
```

Comment:
This line is >90 characters. That's the (undocumented) limit we've been sticking to across this repo. Could you break this line into two?